weave/trace/thread_host.py

```
@@ -0,0 +1,69 @@
+        allow_methods=["*"],
+        allow_headers=["*"],
+    )
+
+    @app.get("/thread/schema")
+    async def thread_spec():
+        return req_type.model_json_schema()
+
+    # TODO: ProjectID should be configurable
+    @app.post("/thread/run")
+    async def thread(item: req_type):
+        with weave.attributes(dict(thread_id=item.thread_id)):
```

Ruff (0.8.2)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)